docker/django/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.4
+
+ENV PYTHONUNBUFFERED 0
+
+RUN apt-get update \
+    && apt-get install -y gdal-bin python-gdal python3-gdal
+
+# Let's install git packages from requirements.txt here
+RUN mkdir -p /src
+
+COPY requirements.txt /src/
+
+RUN pip install --upgrade pip \
+    && pip install --upgrade typing \
```

Comment:
Why add typing? Especially through dockerfile?
Python 3.5 (and thus 3.6) will bring it along also…